### ISSUE TITLE
[Adaptive Cards-Schema Explorer]: Some of the Content is missing from the Adaptive Cards when Text Spacing is applied on the webpage.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7889]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)
Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/%5D(https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))

Pre-Requisites:
Apply Text Spacing on the Adaptive Card Webpage.

Repro Steps:
1. Open above URL with V- credential.
2. Navigate to 'Schema Explorer' link using tab key and press enter key to activate it.
3. Navigate to "TextBlock" link present under CardElements from the left navigation using the tab key and activate it using Enter key.
4. Apply Text Spacing for the Webpage using "Text Spacing" bookmark.
5. Navigate to this content "wrap: false Lorem ipsum dolor sit amet, consectetur...." present under the adaptive card.
6. Observe the issue here.

Actual Result:
"Consectetur" text is missing from the adaptive card, when applied Text spacing on the webpage.

Expected Result:
"Consectetur" text should not get missed from the adaptive card, when applied Text spacing on the webpage.and nothing should miss on the page when applied Text Spacing on the webpage.

User Impact:
Low Vision Keyboard users will be impacted if content missing on the page as they miss some of the information on the page.

MAS Reference:
https://liquid.microsoft.com/Web/Object/Read/ms.accessibility/Requirements/01.04.12

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

<img width="558" alt="Text spacing issue" src="https://user-images.githubusercontent.com/104482612/190590127-8d0630e4-1f72-4cfd-9713-9e782987686c.png">

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by anna-dingler** [comment]
Hi @Susmitha474, can you give more details for how you apply text spacing?

**Comment by Susmitha474** [comment]
> Hi @Susmitha474, can you give more details for how you apply text spacing?

@anna-dingler please refer prerequisite I have added the process over there.

**Comment by anna-dingler** [comment]
Hi @Susmitha474. Thank you for adding additional details on Stylus!

For the specific example you pointed out, we are showcasing an example of the `wrap` property in our schema. With `wrap` set to `false`, the content is intended to only stay on one line. This is why an extra word is cut off after text spacing is applied, but it does not impact the meaning of the card as we are intending on cutting off the text early.

Let me know if you have any questions!

**Comment by vagpt** [comment]
As per the above comment closing this issue.
